static/scss/generic/_vertical-divider.scss

```diff
@@ -31,19 +31,43 @@
         height: 100%;
         width: 1px;
         border-left: 1px dotted $warm-grey;
+
+        @media only screen and (max-width: $breakpoint-medium) {
+          display: none;
+        }
       }
     }
 
-    > article:last-of-type{
+    > *:last-of-type{
       padding-right: 0;
 
       &:before {
         border-left: 0;
       }
     }
 
-    > article:first-of-type {
+    > *:first-of-type {
       padding-left: 0;
     }
   }
+
+  .horizontal-divider {
+    position: relative;
+
+    > * {
```

Comment:
Again it's good practice to target a class name to stop unexpected side effects and gives you freedom to target some but not all children.